lib/WP_Auth0_Options.php

```diff
@@ -129,92 +134,124 @@ public function get_logout_url() {
 		return add_query_arg( 'action', 'logout', site_url( 'wp-login.php', 'login' ) );
 	}
 
+	/**
+	 * Get as a string-separated string and parse to array
+	 *
+	 * @return array
+	 */
+	public function get_lock_connections() {
+		$connections = $this->get( 'lock_connections' );
```

Comment:
why is the `lock_connections` a "string,of,connections"? It seems you are always manipulating arrays. Can it be defined as an array to avoid all this coming and going from one structure to the other?

Reply:
The field in on the settings page is just a text field so comma-separated (or line break or pipe or whatever) is about the best we can do without a UI to force multiple, separate values. This is basically to handle user input